### ISSUE TITLE
Branch 3.10: Backend > Location > Edit: Delete button is missing?

### ISSUE THREAD

**Issue by jeroendesloovere** [forkcms/forkcms#1411]
### Problem
When visiting Backend > Location > Edit: Delete is missing, the following code is gone?
```
{option:showLocaleDelete}
<a href="{$var|geturl:'delete'}&amp;id={$id}{$filterQuery}" data-message-id="confirmDelete" class="askConfirmation button linkButton icon iconDelete">
    <span>{$lblDelete|ucfirst}</span>
</a>
<div id="confirmDelete" title="{$lblDelete|ucfirst}?" style="display: none;">
    <p>
        {$msgConfirmDelete}
     </p>
</div>
{/option:showLocaleDelete}
```
> [File](https://github.com/forkcms/forkcms/blob/3.10/src/Backend/Modules/Locale/Layout/Templates/Edit.tpl#L49)

**Comment by WouterSioen** [comment]
nope, it's caused by the options that aren't parsed anymore. I'm working on this (but it goes rather slow, sorry)

**Comment by WouterSioen** [comment]
fixed in 4.0.0
